### PR TITLE
Issue name : Improving the navigation bar UI

### PR DESCRIPTION
**Pull request by Kritika0035**
Fixed the navigation bar of the UI, issue no. 289.
SSOC member.
Screenshot (198)

### CODE REVIEW

**Comment by myhacktech** [comment]
try to improve this

**Comment by Kritika0035** [comment]
The picture isn't visible

**Comment by myhacktech** [comment]
> The picture isn't visible

Screenshot (706)

**Comment by Kritika0035** [comment]
okay I will fix it

**Review by Susmita-Dey — COMMENTED** [review]
Any updates?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
@Kritika0035 Your branch has merge conflicts. Kindly fix it.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

File: css/style.css [hunk 1]
```diff
@@ -164,6 +165,7 @@ header {
 
 .login-button:hover {
   box-shadow: inset 130px 0 0 0 #0da1dc;
+  transition: 0.5s ease-in-out;
```
Comment (AliReza1083) [inline css/style.css:168]: If you want to code like a professional, there is no need to code like this:
```diff
- transition: 0.3s ease-in-out;
+ transition: .3s ease-in-out;
```
Overall looking Great.

**Review by AliReza1083 — COMMENTED** [review]
I reviewed your codes, and I came up with this.
Wish you all the best.

**Pull request closed without merging**